packages/components/src/components/input/input.lite.tsx

```diff
@@ -0,0 +1,67 @@
+import { onMount, Show, useMetadata, useStore } from "@builder.io/mitosis";
+import { DBInputState, DBInputProps, iconVariants } from "./model";
+import { DBIcon } from '../icon';
+import "./input.scss";
+
+useMetadata({
+  isAttachedToShadowDom: false,
+  component: {
+    includeIcon: false,
+    properties: [],
+  },
+});
+
+export default function DBInput(props: DBInputProps) {
+  const state = useStore<DBInputState>({});
+
+  onMount(() => {
+    if (props.stylePath) {
+      state.stylePath = props.stylePath;
+    }
+  });
+
+  return (
+		<div
+			class={
+				'db-input' +
```

Comment:
```suggestion
				'db-input ' +
```
My understanding of this would be that there wouldn't be a space in between `db-input` and another class added by className if we don't add one here at the end (or a more complicated solution within the next line).